<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>console</class>
 <widget class="QWidget" name="console">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>807</width>
    <height>900</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>750</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>console</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#console
{
    border-image: url(:/page_preview/Resources/BackGround2.png);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget_3" native="true">
     <property name="minimumSize">
      <size>
       <width>800</width>
       <height>750</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="styleSheet">
           <string notr="true">font-size: 16px;
font-family: Microsoft YaHei;</string>
          </property>
          <property name="text">
           <string>欢迎你</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="PB_library">
          <property name="styleSheet">
           <string notr="true">width: 120px;
height: 28px;
background: #E0463E;
border-radius: 4px;
font-size: 12px;
font-family: Microsoft YaHei;
font-weight: 400;
color: #FEFEFE;
line-height: 60px;</string>
          </property>
          <property name="text">
           <string>图书管理</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="PB_record">
          <property name="styleSheet">
           <string notr="true">width: 120px;
height: 28px;
background: #286DB5;
border-radius: 4px;
font-size: 12px;
font-family: Microsoft YaHei;
font-weight: 400;
color: #FEFEFE;
line-height: 60px;</string>
          </property>
          <property name="text">
           <string>(借阅/归还)记录</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="PB_user">
          <property name="styleSheet">
           <string notr="true">QPushButton#PB_user
{
width: 120px;
height: 28px;
background: #31B880;
border-radius: 4px;
font-size: 12px;
font-family: Microsoft YaHei;
font-weight: 400;
color: #FEFEFE;
line-height: 60px;
}
QPushButton#PB_user:hover
{
width: 120px;
height: 28px;
background: #31B880;
border-radius: 4px;
font-size: 12px;
font-family: Microsoft YaHei;
font-weight: 400;
color: #000000;
line-height: 60px;
}

QPushButton#PB_user:pressed
{
width: 120px;
height: 28px;
background: #31B880;
border-radius: 4px;
font-size: 12px;
font-family: Microsoft YaHei;
font-weight: 400;
color: #FEFEFE;
line-height: 60px;
}</string>
          </property>
          <property name="text">
           <string>用户列表</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="minimumSize">
         <size>
          <width>770</width>
          <height>700</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="page">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="page_preview" name="page1_preview" native="true">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_2">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="Borrowed_Manager" name="widget" native="true">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_3">
         <layout class="QGridLayout" name="gridLayout_3">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="User_Manager" name="widget_2" native="true"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>page_preview</class>
   <extends>QWidget</extends>
   <header>page_preview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Borrowed_Manager</class>
   <extends>QWidget</extends>
   <header>borrowed_manager.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>User_Manager</class>
   <extends>QWidget</extends>
   <header>user_manager.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
